.eslintrc

```diff
@@ -253,6 +256,91 @@
     "space-infix-ops": 2, // http://eslint.org/docs/rules/space-infix-ops
     "space-in-parens": [2, "never"], // http://eslint.org/docs/rules/space-in-parens
     "spaced-comment": [2, "always"], // http://eslint.org/docs/rules/spaced-comment
-    "arrow-spacing": [2, { "before": true, "after": true }] // https://eslint.org/docs/rules/arrow-spacing
+    "arrow-spacing": [2, { "before": true, "after": true }], // https://eslint.org/docs/rules/arrow-spacing
+
+    /**
+    * Additional rules to enable one by one
+    */
+    // "array-bracket-spacing": ["error", "never"],
+    // "array-callback-return": ["error", { "allowImplicit": true }],
+    // "arrow-body-style": ["error", "as-needed", { "requireReturnForObjectLiteral": false }],
+    // "arrow-parens": [ "error", "as-needed", { "requireForBlockBody": true }],
+    // "block-spacing": ["error", "always"],
+    // "computed-property-spacing": ["error", "never"],
+    // "dot-location": ["error", "property"],
+    // "function-paren-newline": ["error", "multiline"],
+    // "import/export": "error",
```

Comment:
@aldeed Do these `import` rules come from benmosher's `eslint-plugin-import` plugin? https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules

Reply:
Yes. The plugin packages all begin with `eslint-plugin-` in the package.json, and then the part of the name after `eslint-plugin-` is referenced in the "plugins" array in this file. Any rule in `import/` namespace is from the import plugin package, etc.

I pulled in the rules Airbnb uses for `import/`, but then changed/removed a few to match Meteor/Reaction conventions. These will mostly help catch consistency things (like order of import statements) and potential issues (like mutable exports).

I added the `eslint-import-resolver-meteor` package, too, which makes it understand Meteor's way and be able to resolve absolute import paths.